app/models/foreman_tasks/task/dynflow_task.rb

```diff
@@ -104,28 +94,27 @@ def get_humanized(method)
     def self.consistency_check
       fixed_count = 0
       logger = Foreman::Logging.logger('foreman-tasks')
-      self.running.each do |task|
+      running.each do |task|
         begin
           changes = task.update_from_dynflow(task.execution_plan.to_hash)
           unless changes.empty?
             fixed_count += 1
-            logger.warn("Task %s updated at consistency check: %s" % [task.id, changes.inspect])
+            logger.warn('Task %s updated at consistency check: %s' % [task.id, changes.inspect])
           end
         rescue => e
           Foreman::Logging.exception("Failed at consistency check for task #{task.id}", e, :logger => 'foreman-tasks')
         end
       end
-      return fixed_count
+      fixed_count
     end
 
     def self.new_for_execution_plan(execution_plan_id, data)
-      self.new(:external_id => execution_plan_id,
-               :state => data[:state].to_s,
-               :result => data[:result].to_s)
+      new(:external_id => execution_plan_id,
+          :state => data[:state].to_s,
+          :result => data[:result].to_s)
     end
 
-    private
-
+    private_class_method :model_name
```

Comment:
I'm not sure we want this method to actually be private? Based on the history, seems this should be used by external code (routes generation)